libs/docs/platform/search-field/examples/platform-search-field-basic-example.component.ts

```diff
@@ -45,6 +45,69 @@ export class PlatformSearchFieldBasicExampleComponent {
         },
         {
             value: 'Raspberry'
+        },
```

Comment:
Maybe instead of hardcoded variants use generated with the help of new Array(n)?

Reply:
done